v2/fleet-local/control/flight-director@.service

```diff
@@ -2,16 +2,18 @@
 Description=Flight Director @ %i
 # it's implied that chronos & marathon require mesos
 # look here: https://github.com/behance/mesos-systemd/tree/master/v2/fleet
-After=docker.service chronos.service marathon.service
+After=docker.service chronos@%i.service marathon@%i.service
```

Comment:
Yea, this one is important, as we stopped enabling chronos on stage and prod bc it caused problems.